e2e/libs/node/node.py

```diff
@@ -1,26 +1,103 @@
-from kubernetes import client
-
-class Node:
+from utility import globals
+class Nodes:
```

Comment:
Remember to update [other imports](https://github.com/longhorn/longhorn-tests/blob/fc211a88b8/e2e/libs/keywords/volume_keywords.py#L4).

Reply:
Thank you. I have check whole project and correct the imported class name.